nrf70_bm_lib/include/nrf70_tx_pwr_ceil_dk.h

```diff
@@ -7,24 +7,34 @@
 
 /** @file
  * @brief nRF7002 DK TX power ceiling configuration.
- */
-
-/*
+ *
  * The configuration applies to the nRF7002 DK board targets, as well as
  * to board targets for nRF7001 and nRF7000 IC variants emulated on the
  * nRF7002 DK.
- */
-
-/* The following macros are normally auto-generated from DeviceTree
+ *
+ * The following macros are normally auto-generated from DeviceTree
  * when building the nRF70 driver in a Zephyr environment. They are
  * copied here as standard preprocessor directives for ease of porting.
+ *
+ * The values are represented in 1 dB increments.
+ * @addtogroup nrf70_bm_txpower nRF70 Bare Metal library
  */
+
+/** @brief Maximum power for 2.4 GHz DSSS. */
 #define MAX_PWR_2G_DSSS 0x54
+/** @brief Maximum power for 2.4 GHz MCS0. */
 #define MAX_PWR_2G_MCS0 0x40
+/** @brief Maximum power for 2.4 GHz MCS7. */
 #define MAX_PWR_2G_MCS7 0x40
+/** @brief Maximum power for 5 GHz low band (5.15-5.25 GHz) MCS0. */
 #define MAX_PWR_5G_LOW_MCS0 0x24
+/** @brief Maximum power for 5 GHz low band (5.15-5.25 GHz) MCS7. */
```

Comment:
there is also a question about the meaning on MCS0/7, how is that used?e.g., MCS3, how does it map?

Reply:
@kspraveeen

Reply:
FW has the capability to interpolate TX power for MCS other than 0,7. 
However, same Tx power is used for all data rates as the value corresponding to MCS0,7 are same.